mlflow/tracking/fluent.py

```diff
@@ -409,7 +409,8 @@ def get_artifact_uri(artifact_path=None):
 
 
 def search_runs(experiment_ids=None, filter_string="", run_view_type=ViewType.ACTIVE_ONLY,
-                max_results=SEARCH_MAX_RESULTS_PANDAS, order_by=None):
+                max_results=SEARCH_MAX_RESULTS_PANDAS, order_by=None,
+                columns_to_whitelist=None):
```

Comment:
It would be nice to add tests for this

Reply:
There are no logic in the functions (just forward the argument to client through _get_paginated_runs). Do you agree to not test that ?